interface/forms/group_attendance/save.php

```diff
@@ -0,0 +1,76 @@
+<?php
+/**
+ * interface/forms/group_attendance/save.php
+ *
+ * Copyright (C) 2016 Shachar Zilbershlag <shachar058@gmail.com>
+ * Copyright (C) 2016 Amiel Elboim <amielboim@gmail.com>
+ *
+ * LICENSE: This program is free software; you can redistribute it and/or
+ * modify it under the terms of the GNU General Public License
+ * as published by the Free Software Foundation; either version 3
+ * of the License, or (at your option) any later version.
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+ * GNU General Public License for more details.
+ * You should have received a copy of the GNU General Public License
+ * along with this program. If not, see <http://opensource.org/licenses/gpl-license.php>;.
+ *
+ * @package OpenEMR
+ * @author  Shachar Zilbershlag <shachar058@gmail.com>
+ * @author  Amiel Elboim <amielboim@gmail.com>
+ * @link    http://www.open-emr.org
+ */
+require_once("functions.php");
+
+$fake_register_globals=false;
+$sanitize_all_escapes=true;
```

Comment:
When you are running a script that will do something, such as this, then need globals.php (this does all the authentication and standard variable setting). Should not include it in libraries.